models/api.models.js

```diff
@@ -33,3 +33,20 @@ exports.selectArticleById = (article_id) => {
     }
   });
 };
+
+exports.selectCommentsByArticle = (article_id) => {
+  return db
+    .query(
+      "SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC",
+      [article_id]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0) {
```

Comment:
Is it an error if a valid article has no comments on it? What about new articles as they wouldn't have any comments on them.